2022/32xxx/CVE-2022-32160.json

```diff
@@ -1,18 +1,83 @@
 {
-    "data_type": "CVE",
-    "data_format": "MITRE",
-    "data_version": "4.0",
-    "CVE_data_meta": {
-        "ID": "CVE-2022-32160",
-        "ASSIGNER": "cve@mitre.org",
-        "STATE": "RESERVED"
-    },
-    "description": {
-        "description_data": [
-            {
-                "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+  "CVE_data_meta" : {
+    "ASSIGNER" : "vulnerabilitylab@mend.io",
+    "ID" : "CVE-2022-32160",
+    "STATE" : "PUBLIC",
+    "DATE_PUBLIC" : "Jun 1, 2022, 4:32:50 AM",
+    "TITLE" : "Bolt - Reflected XSS"
+  },
+  "affects" : {
+    "vendor" : {
+      "vendor_data" : [ {
+        "vendor_name" : "core",
+        "product" : {
+          "product_data" : [ {
+            "product_name" : "core",
+            "version" : {
+              "version_data" : [ {
+                "version_value" : "5.1.3",
+                "version_affected" : ">="
+              }, {
+                "version_value" : "5.1.5",
+                "version_affected" : "<="
+              } ]
             }
-        ]
+          } ]
+        }
+      } ]
     }
+  },
+  "credit" : [ {
+    "lang" : "eng",
+    "value" : "Mend Vulnerability Research Team (MVR)"
+  } ],
+  "data_format" : "MITRE",
+  "data_type" : "CVE",
+  "data_version" : "4.0",
+  "description" : {
+    "description_data" : [ {
+      "lang" : "eng",
+      "value" : "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
```

Comment:
Hello @danielelkabes,

Please update the description here with the information about the Vulnerability.

Thanks,
CVE Content Team